wp_insert_rows.php

```diff
@@ -17,18 +18,21 @@
 	*
 	*  }
 	*
-	*  wp_insert_rows($insert_arrays);
+	*  wp_insert_rows($insert_arrays, $wpdb->tablename, true, "column_name_of_Primary_key");
```

Comment:
This is a new usage case and Instead of deleting old one, it should has a separate usage example.